po/de.po

```diff
@@ -666,9 +660,8 @@ msgstr ""
 "Für diejenigen, die nicht mit git vertraut sind, hat das <span class="
 "\"gnucash\">GnuCash</span>-Wiki eine maßgeschneiderte englischsprachige <a "
 "href='https://wiki.gnucash.org/wiki/Git'>Anleitung</a> für den Quelltext des "
-"Programms. \n"
-"Um die Dokumentation zu bekommen, muß man der Anleitung folgen und dabei "
-"<i>checkout gnucash</i> ersetzen durch <i>checkout gnucash-docs</i> wie hier:"
+"Programms. Um die Dokumentation zu bekommen, muß man der Anleitung folgen "
+"und dabei 'checkout gnucash' ersetzen durch 'checkout gnucash-docs' wie hier:"
```

Comment:
Dies ist so ein Fall, wo man eher den englischen Text anpassen sollte. Im Gegensatz zum Original war die deutsche Version –ja, lang ist's her– mal einheitlich mit den Auszeichnungen.
Siehe https://wiki.gnucash.org/wiki/Website_Maintenance und als Beispiele für heutigen (HTML5) Gebrauch die Release Notes in news, z.B. https://github.com/Gnucash/gnucash-htdocs/blob/master/news/210328-4.5.news

Im Moment habe ich nicht die Zeit, die Quelltexte in der Hinssicht mal durchzusehen und ein gutes Dutzend Übersetzungen zu pachen. Falls Du  dazu Lust hast, darfst Du das gerne machen.

Reply:
schau ich mir mal an